packages/api-client/src/api/getCategory/defaultQuery.ts

```diff
@@ -0,0 +1,61 @@
+import gql from 'graphql-tag';
+
+export default gql`
+  query collections {
+	collections{
+      totalItems
+      items {
+        id
+        createdAt
+        updatedAt
+        languageCode
+        name
+        breadcrumbs {
+          id
+          name
+          slug
+        }
+        description
+        position
+        slug
+        featuredAsset {
+          id
+          name
+        }
+        assets {
+          id
+          name
+        }
+        parent {
+          id
+          name
+        }
+        children {
+          id
+          name
+        }
+        filters {
+          code
+        }
+        translations {
+          id
+          createdAt
+          updatedAt
+          languageCode
+          name
+          slug
+          description
+        }
+        productVariants {
```

Comment:
If the intention is to list all the variants in a collection, I recommend using the `search` query with the corresponding `collectionId` or `collectionSlug` input argument. This is a _much_ more efficient way to get that data.